setup.py

```diff
@@ -152,7 +152,6 @@
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: POSIX :: Linux",
         "Operating System :: MacOS :: MacOS X",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
Feel free to add 3.10, 3.11, 3.12 or whatever are supported by PyTorch

Reply:
It's not clear if Python 3.12 is supported by PyTorch so skipped it for now.